healthstatus.py

```diff
@@ -191,17 +191,14 @@ async def dowork(rec: MemoryObjectReceiveStream[Dandiset]) -> None:
             }
             test_summaries = {tn: TestSummary(tn) for tn in TEST_NAMES}
             for r in all_reports:
+                passed, failed, timedout = r.combined_counts()
+                asset_qtys[Outcome.PASS] += passed
+                asset_qtys[Outcome.FAIL] += failed
+                asset_qtys[Outcome.TIMEOUT] += timedout
+                dandiset_qtys[Outcome.PASS] += bool(passed)
```

Comment:
I think this is wrong logic which would report dandiset if any asset passed any test, isn't it so?  don't we need smth like

```suggestion
                dandiset_qtys[Outcome.PASS] += bool(not failed and not timedout)
```

?

Reply:
Right, I missed that you said the number of passing Dandisets should be "where all assets pass all tests".